client/src/pages/Landing/WhatOurUsersSay/WhatOurUsersSay.module.scss

```diff
@@ -37,19 +33,13 @@
 }
 
 @media screen and (min-width: 760px) {
-  .single_feedback_container {
-    float: left;
-    width: 50%;
-    padding-right: 2rem;
-    background: #f2f4f7;
-  }
-  .feedback_container {
-    background: #f2f4f7;
+  .clearfix {
+    grid-template-columns: 1fr 1fr;
   }
 }
 
 @media screen and (min-width: 992px) {
-  .single_feedback_container {
-    width: 33.3%;
+  .clearfix {
+    grid-template-columns: 1fr 1fr 1fr;
   }
 }
```

Comment:
can be achieved with:     

`grid-template-columns: repeat(auto-fit, minmax(min(100%, 260px), 1fr));
`

Reply:
This looks like magic!
Can you explain how it works?

Reply:
260px results in a 4 column layout on the big screen. I used 300px instead